frontend/package.json

```diff
@@ -27,7 +27,7 @@
     "react-scripts": "^5.0.0",
     "redux": "^4.1.2",
     "redux-persist": "^6.0.0",
-    "typescript": "~4.1.5",
+    "typescript": "~4.5.4",
```

Comment:
Upgrade required in order to type correctly `Int.DisplayNames` (used to get language names)